docker-maintainer/README.md

```diff
@@ -0,0 +1,94 @@
+* Scripts for reading the log files of the webserver and the judgedaemons.
+* A script to create a dummy DOMjudge user and submit all test submissions.
+
+This container does not include:
+
+* MySQL / MariaDB; the [MariaDB](https://hub.docker.com/r/_/mariadb/) Docker container does this better than we ever could.
+* The DOMjudge source code itself. The [DOMjudge repository](https://github.com/domjudge/domjudge) should be cloned on the target machine somewhere and a volume should be created for it.
+
+## Using the image
+
+This image is available on the [Docker Hub](https://hub.docker.com) as `domjudge/domjudge-maintainer`.
+
+Before starting the container, make sure you have a MySQL / MariaDB database somewhere. The easiest way to get one up and running is to use the [MariaDB](https://hub.docker.com/r/_/mariadb/) Docker container:
+
+```bash
+docker run -it --name dj-mariadb -e MYSQL_ROOT_PASSWORD=rootpw -e MYSQL_USER=domjudge -e MYSQL_PASSWORD=djpw -e MYSQL_DATABASE=domjudge -p 13306:3306 mariadb
+```
+
+This will start a MariaDB container, set the root password to `rootpw`, create a MySQL user named `domjudge` with password `djpw` and create an empty database named `domjudge`. It will also expose the server on port `13306` on your local machine, so you can use your favorite MySQL GUI to connect to it. If you want to save the MySQL data after removing the container, please read the [MariaDB](https://hub.docker.com/r/_/mariadb/) Docker Hub page for more information.
+
+Now you can run DOMjudge itself using the following command:
+
+```bash
+docker run -v [path-to-domjudge-checkout]:/domjudge --link dj-mariadb:mariadb -it -e MYSQL_HOST=mariadb -e MYSQL_USER=domjudge -e MYSQL_DATABASE=domjudge -e MYSQL_PASSWORD=djpw -e   MYSQL_ROOT_PASSWORD=rootpw -p 12345:80 --name domjudge --privileged domjudge/domjudge-maintainer
+```
+
+Make sure you replace `[path-to-domjudge-checkout]` with the path to your local DOMjudge checkout.
+
+On macOS and Windows you need to add `-v /sys/fs/cgroup:/sys/fs/cgroup:ro` to this command (somewhere after `run` and before `domjudge/domjudge-maintainer`) to get the cgroups to work correctly.
+
+The above command will start the container, set up DOMjudge for a maintainer install, set up the database and create a chroot to be used by the judgedaemons. It will then start nginx, PHP-FPM and two judgedaemons using supervisord.
+
+You can now access the web interface on [http://localhost:12345/](http://localhost:12345/). Use username `admin` and password `admin` to log in. Note that for DOMjudge 6.0.0 and higher the webserver configuration will be set up such that the debug front controller will be used.
+
+### Environment variables
+
+The following environment variables are supported by the container:
+
+* `CONTAINER_TIMEZONE` (defaults to `Europe/Amsterdam`): allows you to change the timezone used inside the container.
+* `PHP_UPLOAD_MAX_FILESIZE` (defaults to `10M`): set the maximum upload filesize for PHP.
```

Comment:
10M is quite low for problems nowadays. Is there any harm by increasing it to 64M?

Reply:
I see no harm, so I'll increase it.

Reply:
It was already 100M as this was an outdated diff :P